### PR TITLE
Create using-minifinder

### PR DESCRIPTION
**Pull request by bennewi**

### CODE REVIEW

File: articles/advanced/using-minifinder [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: Using the Minifinder tool for Eclipse
+order: 41
```
Comment (jouni) [inline articles/advanced/using-minifinder:3]: Articles under the Advanced section aren't ordered.

```suggestion
```

File: articles/advanced/using-minifinder [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+= Minifinder for Eclipse 
+:toclevels: 2
+
+pass:[<!-- vale Vaadin.ProductName = NO -->]
```
Comment (jouni) [inline articles/advanced/using-minifinder:9]: Why is this rule disabled?

File: articles/advanced/using-minifinder [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+==== Running the analyzer
+Click the button in the toolbar that appeared when you installed the plugin. The tool is set up with the defaults that work with over 97% of all Vaadin 8 applications so pressing the [OK] button should be all you need to do.
+
+A progress bar should appear that shows the progress of the analysis.
+
+==== Interpreting the results
+When the progress bar reaches 100% and disappears, the analysis will be ready and you can view the output. Minifinder for Eclipse puts the output in the Eclipse Console. 
+
+Window > Show View > Console
```
Comment (jouni) [inline articles/advanced/using-minifinder:50]: Use the `menuseq` macro.

File: articles/advanced/using-minifinder [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+The stats for method and constructor invocations include an indication of “coverage.” In general, the higher the coverage rate, the better. Coverage refers to a situation that is known and is dealt with with high confidence that no further developer work will be needed. Said otherwise:
+- The transformation has been established for this declaration and the transformation tool contains a rule that handles it, and
+- The transformation requires no change, or
+- The transformation requires a change and this change is made by the tool with no manual effort required
+
+Coverage assumes that the modernized application will use Vaadin 24 Classic Components and a handful of utility methods that help bridge the gap in behavior between Vaadin 8 and Vaadin 24. 
+
+
```
Comment (jouni) [inline articles/advanced/using-minifinder:63]: Add a discussion ID.

File: articles/advanced/using-minifinder [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+The stats for method and constructor invocations include an indication of “coverage.” In general, the higher the coverage rate, the better. Coverage refers to a situation that is known and is dealt with with high confidence that no further developer work will be needed. Said otherwise:
+- The transformation has been established for this declaration and the transformation tool contains a rule that handles it, and
+- The transformation requires no change, or
+- The transformation requires a change and this change is made by the tool with no manual effort required
+
+Coverage assumes that the modernized application will use Vaadin 24 Classic Components and a handful of utility methods that help bridge the gap in behavior between Vaadin 8 and Vaadin 24.
```
Comment (jouni) [inline articles/advanced/using-minifinder:61]:
```suggestion
Coverage assumes that the modernized application uses Classic Components and a handful of utility methods that help bridge the gap in behavior between Vaadin 8 and later versions. 
```

File: articles/advanced/using-minifinder [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+One thing you might consider is to close any project in your workspace of which you know that it does not contain any Vaadin 8 references. This will prevent Minifinder from analyzing any code of which you know that it is not relevant to Vaadin 8 and you might shave off a few minutes from the analysis.
+
+
+==== Running the analyzer
+Click the button in the toolbar that appeared when you installed the plugin. The tool is set up with the defaults that work with over 97% of all Vaadin 8 applications so pressing the [OK] button should be all you need to do.
+
+A progress bar should appear that shows the progress of the analysis.
+
+==== Interpreting the results
+When the progress bar reaches 100% and disappears, the analysis will be ready and you can view the output. Minifinder for Eclipse puts the output in the Eclipse Console.
```
Comment (jouni) [inline articles/advanced/using-minifinder:48]:
```suggestion
When the progress bar reaches 100% and disappears, the analysis is ready and you can view the output. Minifinder for Eclipse puts the output in the Eclipse Console. 
```

File: articles/advanced/using-minifinder [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+Always make sure that you have the most up to date version of Minifinder because it will contain the latest set of transformation rules that are available with the V8 Upgrade Automation.
+
+==== Ensure your application projects are ready to be analyzed
+Minifinder depends on the Eclipse environment to resolve all names in your code with their corresponding types and packages in which they are declared. Eclipse is very robust and can resolve names in sources that contain errors, so Minifinder may work even if your project has hundreds of Maven errors or Java errors. Having said that, the fewer errors you have the better so we would encourage you to ensure your project is free of red lines before launching Minifinder.
+
+One thing you might consider is to close any project in your workspace of which you know that it does not contain any Vaadin 8 references. This will prevent Minifinder from analyzing any code of which you know that it is not relevant to Vaadin 8 and you might shave off a few minutes from the analysis.
```
Comment (jouni) [inline articles/advanced/using-minifinder:39]:
```suggestion
One thing you might consider is to close any project in your workspace of which you know that it does not contain any Vaadin 8 references. This prevents Minifinder from analyzing any code of which you know that it is not relevant to Vaadin 8 and you might shave off a few minutes from the analysis.
```

File: articles/advanced/using-minifinder [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+pass:[<!-- vale Vaadin.ProductName = NO -->]
+
+Minifinder for Eclipse is a tool built by Vaadin to help Vaadin 8 developers understand how well their applications can be transformed by Vaadin’s semi-automated modernization service called https://vaadin.com/vaadin-8-upgrade-automation-service[V8 Upgrade Automation]. 
+
+Minifinder works by parsing the Java code of your projects, resolving as many of your application’s references to Vaadin libraries as it can, and outputting the high-level results in the Console of your Eclipse IDE. You can copy the report and share with your colleagues or your Vaadin experts in order to discuss the benefits and make an informed decision on how to move forward.
+
+
+=== Limitations
+To make this application as easy as possible to be used at companies with strict security policies regarding installing tools in their network, the application is limited in what it can do:
+- Minifinder runs completely locally and will not connect to the Internet
```
Comment (jouni) [inline articles/advanced/using-minifinder:18]:
```suggestion
- Minifinder runs completely locally and doesn't connect to the Internet
```

File: articles/advanced/using-minifinder [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+- It will not change your code
```
Comment (jouni) [inline articles/advanced/using-minifinder:19]:
```suggestion
- It doesn't change your code
```

File: articles/advanced/using-minifinder [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+- Minifinder will write all output to the console and not write to disk
```
Comment (jouni) [inline articles/advanced/using-minifinder:20]:
```suggestion
- Minifinder writes all output to the console and not to disk
```

**Review by jouni — COMMENTED** [review]
Some quick drive-by remarks.

File: articles/advanced/using-minifinder [hunk 11]
```diff
@@ -0,0 +1,67 @@
+---
+title: Using the Minifinder tool for Eclipse
```
Comment (tarekoraby) [inline articles/advanced/using-minifinder:2]: I suggest using a title descriptive of the use case. No one outside Vaadin will know what "Minifinder" is.

**Review by Mikaelsu — APPROVED** [review]
Pushed my review commit earlier.

**Pull request merged**